spec/features/show_page_spec.rb

```diff
@@ -28,6 +29,22 @@
       expect(ids_in_table).to match_array(order_ids)
     end
 
+    it "displays the first page by default, other pages when specified where \
+      the total number of records don't exceed pagination per page limit" do
```

Comment:
What do you think about this phrasing? I think it reflects the situation a bit better:

```suggestion
    it "displays all results when the total number of records \
        doesn't exceed pagination per page limit" do
```

Also, since each has a context ("where exceeding (or not) the page limit"), perhaps these would read better if they used `context`?

```ruby
context "when the total number of records exceeds the pagination limit" do
  it "displays the first page by default, other pages on request"
end
context "when the total number of records does not exceed the pagination limit" do
  it "displays all records"
end
```